label_db.py

```diff
@@ -84,11 +87,20 @@ def _add_rows_for_labels(self, img_id, labels, flip=False):
       c.execute("insert into labels (img_id, x, y) values (?, ?, ?)", (img_id, x, y,))
     self.conn.commit()
 
+  def clean_db(self):
+    c = self.conn.cursor()
+    c.execute("delete from labels")
+    c.execute("delete from imgs")
+    self.conn.commit()
 
 if __name__ == "__main__":
   import argparse
   parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
   parser.add_argument('--label-db', type=str, default="label.db")
+  parser.add_argument('--clean-db', type=str, default="label.db")
   opts = parser.parse_args()
   db = LabelDB(label_db_file=opts.label_db)
+  db.create_if_required()
+  if(sys.argv[1] == '--clean-db'):
```

Comment:
this is not how argparse works. see https://docs.python.org/3/library/argparse.html

Reply:
( that aside, it's simpler to clean by just removing the file on disk )